### PR TITLE
Update feast transformer docs

### PR DESCRIPTION
**Pull request by sivanantha321**
Updated feast transformer example

## Proposed Changes 

- docs for example in https://github.com/kserve/kserve/pull/2780
-
-

### CODE REVIEW

File: docs/modelserving/v1beta1/transformer/feast/README.md [hunk 1]
```diff
@@ -1,11 +1,160 @@
...
+## Create the Feast server
+
+### Build Feature Store Initializer docker image
+The feature store initializer is a init container which initializes a new sample feature repository, populate the 
+online store with sample driver data and copies the feature repository to the volume mount.
+
+Checkout the feast code example and under the example directory run the commands as following:
+
+```bash
+docker build -t {username}/feature-store-initializer:latest -f feature_store_initializer.Dockerfile .
```
Comment (yuzisun) [inline docs/modelserving/v1beta1/transformer/feast/README.md:75]: Add a link to the `feature_store_initializer.Dockerfile` file

File: docs/modelserving/v1beta1/transformer/feast/README.md [hunk 2]
```diff
@@ -1,11 +1,160 @@
...
+
+docker push {username}/feature-store-initializer:latest
+```
+
+### Build Feast server docker image
+
+```bash
+docker build -t {username}/feast-server:latest -f feast_server.Dockerfile .
```
Comment (yuzisun) [inline docs/modelserving/v1beta1/transformer/feast/README.md:83]: Add a link to the feast_server.Dockerfile file

File: docs/modelserving/v1beta1/transformer/feast/README.md [hunk 3]
```diff
@@ -36,51 +183,88 @@ docker push {username}/driver-transformer:latest
...
+    ```bash
+    cat <<EOF | kubectl apply -f -
+    apiVersion: "serving.kserve.io/v1beta1"
+    kind: "InferenceService"
+    metadata:
+      name: "sklearn-driver-transformer"
+    spec:
+      transformer:
+        containers:
+          - image: "{username}/driver-transformer:latest"
```
Comment (yuzisun) [inline docs/modelserving/v1beta1/transformer/feast/README.md:200]: We can update the kserve image?

File: docs/modelserving/v1beta1/transformer/feast/README.md (hunk 1 above)
Comment (sivanantha321) [reply docs/modelserving/v1beta1/transformer/feast/README.md:75]: sure

File: docs/modelserving/v1beta1/transformer/feast/README.md (hunk 2 above)
Comment (sivanantha321) [reply docs/modelserving/v1beta1/transformer/feast/README.md:83]: sure

File: docs/modelserving/v1beta1/transformer/feast/README.md (hunk 3 above)
Comment (sivanantha321) [reply docs/modelserving/v1beta1/transformer/feast/README.md:202]: yes, we can update the image

File: docs/modelserving/v1beta1/transformer/feast/README.md [hunk 4]
```diff
@@ -1,11 +1,161 @@
...
+          emptyDir:
+            sizeLimit: 100Mi
+
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: feature-server-service
+spec:
+  type: LoadBalancer
```
Comment (yuzisun) [inline docs/modelserving/v1beta1/transformer/feast/README.md:144]: Does this need to be `LoadBalancer` as it is only accessed in cluster?

File: docs/modelserving/v1beta1/transformer/feast/README.md [hunk 5]
```diff
@@ -1,11 +1,161 @@
...
+    ```{ .bash .no-copy }
+    $ deployment.apps/redis-server created
+    $ service/redis-service created
+    ```
+
+## Create the Feast server
+
+### Build Feature Store Initializer docker image
+The feature store initializer is a init container which initializes a new sample feature repository, populate the 
+online store with sample driver data and copies the feature repository to the volume mount.
```
Comment (yuzisun) [inline docs/modelserving/v1beta1/transformer/feast/README.md:70]: I guess with init container the features are static and the feast server does not refresh when online store is updated, worth noting that but it is fine as an example

File: docs/modelserving/v1beta1/transformer/feast/README.md (hunk 3 above)
Comment (yuzisun) [reply docs/modelserving/v1beta1/transformer/feast/README.md]: image is published `kserve/driver-transformer:latest`

**Comment by yuzisun** [comment]
Thanks @sivanantha321 !

/lgtm
/approve

**Pull request merged**